client/components/single-product-plan/README.md

```diff
@@ -0,0 +1,39 @@
+Single Product Plan
+=======
+
+Single Product Plan is a React component for rendering a box with single product short description and purchase options.
```

Comment:
@tyxla Actually I think this should be:

```single product plan short description```

instead of:

```single product short plan description```

What do you think?

Reply:
Heh, yes, you get the point 😆